lib/pelemay.ex

```diff
@@ -231,7 +240,7 @@ defmodule Analyzer.AFunc do
     end
   end
 
-  def polynomial(ast) do
+  def polynomial_map(ast) do
```

Comment:
I guess that this function polynomial_map needs some tests.
However, if this PR is placed at the position that it is a only small step for covering whole test cases, the testing for polynomial_map may be postponed.